common/persistence/client/factory.go

```diff
@@ -291,80 +291,77 @@ func (f *factoryImpl) NewVisibilityManager(
 			f.logger.Fatal("Creating Pinot advanced visibility manager failed", tag.Error(err))
 		}
 
+		visibilityMgrs := map[string]p.VisibilityManager{
+			"db":    visibilityFromDB,
+			"pinot": visibilityFromPinot,
```

Comment:
instead of hardcoded strings, we can use the constants you defined. AdvancedVisibilityModeES/Pinot